ui/src/app/views/sign-in.tsx

```diff
@@ -100,7 +101,7 @@ export const SignInReact = withWindowSize()(
     }
 
     componentDidMount() {
-      document.body.style.backgroundColor = '#e2e3e5';
+      document.body.style.backgroundColor = colors.light;
```

Comment:
both this and the above are being converted to colors.light, but they are different hex values.  intentional?

Reply:
Yes, intentional for consistency.